### PR TITLE
Add 101._symmetric_tree.md

### PR DESCRIPTION
**Pull request by zmzhouXJTU**

### CODE REVIEW

File: 101._symmetric_tree.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
...
+### Solutions
+
+#### Approach 1: Recursive
+
+* Time complexity : O(n)
+* Space complexity : O(n)
+
+```Python
+class Solution(object):
+    def isSymmetric(self, root):
```
Comment (jovan-wong) [inline 101._symmetric_tree.md]: We usually name a method with `_` in python code style.
I think `is_symmetric` will be better.

File: 101._symmetric_tree.md [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+        """
+        :type root: TreeNode
+        :rtype: bool
+        """
+        if not root:
```
Comment (jovan-wong) [inline 101._symmetric_tree.md]: `if not root:` --> `if root is not None:`?
`if not root:`  will make us confused: whether root is a bool type.

File: 101._symmetric_tree.md [hunk 3]
```diff
@@ -0,0 +1,82 @@
...
+
+#### Approach 1: Recursive
+
+* Time complexity : O(n)
+* Space complexity : O(n)
+
+```Python
+class Solution(object):
+    def isSymmetric(self, root):
+        """
```
Comment (jovan-wong) [inline 101._symmetric_tree.md]:
```python
    """
    xxxxxxxxxxxxxxxx(brief of this method)

    Args:
        root(TreeNode): The root of xxxx

    Returns:
        xxxxxxxxxxxxxxx
        """
```
doc comments in python should be like this.

File: 101._symmetric_tree.md [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+        :type root: TreeNode
+        :rtype: bool
+        """
+        if not root:
+            return True
+        return self.symmetric(root.left, root.right)
+        
+    def symmetric(self, l1, l2):
```
Comment (jovan-wong) [inline 101._symmetric_tree.md]: make l1 and l2 be meaningful names?

Comment (zmzhouXJTU) [reply 101._symmetric_tree.md]: ok，I have revised my code

**Pull request merged**